docker/package/model.py

```diff
@@ -644,13 +646,22 @@ def gen_makefile(self, out):
 
 tezos-baking:
 
+wizard-structure:
+	mv $(CURDIR)/wizard_structure.py $(CURDIR)/wizard_structure.py
+
 tezos-setup-wizard:
 	mv $(CURDIR)/tezos_setup_wizard.py $(CURDIR)/tezos-setup-wizard
 	chmod +x $(CURDIR)/tezos-setup-wizard
 
-install: tezos-baking tezos-setup-wizard
+tezos-voting-wizard:
+	mv $(CURDIR)/tezos_voting_wizard.py $(CURDIR)/tezos-voting-wizard
+	chmod +x $(CURDIR)/tezos-voting-wizard
+
+install: wizard-structure tezos-baking tezos-setup-wizard tezos-voting-wizard
 	mkdir -p $(DESTDIR)$(BINDIR)
+	cp $(CURDIR)/wizard_structure.py $(DESTDIR)$(BINDIR)/wizard_structure.py
```

Comment:
TBH, I'm not sure that it's a good idea to put `*.py` files to the `/usr/bin`.

Seems like there is a proper approach for packaging python applications for Debian, see [this doc](https://wiki.debian.org/Python/LibraryStyleGuide?action=show&redirect=Python%2FPackaging).

However, I don't really insist on doing everything properly in this PR since it's already open for quite a long time, and we'd like to include it into the next release, but please create a follow-up issue:)

Reply:
I agree. I'll create the issue as soon as we merge the PR, to have something to link to